back/app/controllers/web_api/v1/admin_publications_controller.rb

```diff
@@ -33,6 +33,21 @@ def reorder
     end
   end
 
+  # For use by homepage to get status counts of visible top-level (not in folder) admin_publications,
+  # using: .../web_api/v1/admin_publications/status_counts?remove_not_allowed_parents=true&depth=0
+  def status_counts
+    publication_filterer = AdminPublicationsFilteringService.new
+    publications = policy_scope(AdminPublication.includes(:parent))
+    publications = publication_filterer.filter(publications, params)
+
+    # For non-admins: publication_status: :draft count should always be zero, and thus not included in response
```

Comment:
Took me a while to understand this comment. Am I right saying that depending on the filters and status of publications on the platform, some publication status might not be shown in the response?

Reply:
`counts = publications.group(:publication_status).count` will not return a key, nor a value, when 0 records found with any specific `publication_status`.

I guess I'm just pointing out that since non-admins never have permission to see `draft` publications, there will never be `draft` key included in the response.

Reply:
But it also applies to situations where for instance all publications on the platform have the same status. Non-represented status would not appear in the response, right?

Reply:
Yup. I think you're right. I should either make the comment say that, or delete it.